jetbrains-core/src/software/aws/toolkits/jetbrains/core/notification/JetBrainsMinimumVersionChange.kt

```diff
@@ -32,12 +32,12 @@ class JetBrainsMinimumVersionChange : NoticeType {
     override fun isNotificationRequired(): Boolean {
         val appInfo = ApplicationInfo.getInstance()
         val majorVersion = appInfo.majorVersion.toIntOrNull()
+        val minorVersion = appInfo.minorVersion.toIntOrNull()
 
-        majorVersion?.let {
-            return majorVersion < 2020
+        if (majorVersion == null || minorVersion == null) {
+            return true
         }
-
-        return true
+        return majorVersion < 2020 || (majorVersion == 2020 && minorVersion < 2)
```

Comment:
`ApplicationInfo.getInstance().getBuild().getBaselineVersion()` is probably the easier than to do since we can just do `< 202`

Reply:
We should have used this before, way easier, changed